classes/trimmer_decorator.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    name = @nameable.correct_name
+    name = name[0, 9] if name.length > 10
+    name
+  end
+end
```

Comment:
- [ ] Kindly ensure that you update the above `correct_name` method in the way it returns the name with a maximum of `10 characters`. If it's longer it should trim the word. **Right now, it returns the name with **9 characters** instead of **10 characters**

Reply:
**[REQUIRED CHANGES]**


- Hey @Swarzstein, The same issue from my previous comment is applicable here. Please, kindly fix this issue. all you need is to change 9 to 10. Please, kindly fix this issue.